packages/lab/src/skip-link/SkipLink.tsx

```diff
@@ -1,11 +1,14 @@
 import { forwardRef, RefObject } from "react";
-import { Link, LinkProps } from "../link";
+import {
+  InternalLinkProps,
```

Comment:
I understand the intention here but is there a reason we wouldn't keep it simple and just use LinkProps?